src/index.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
```

Comment:
Love that you created folders to hold you files to make it more clean and organized !